app/javascript/bundles/Groceries/components/Groceries.tsx

```diff
@@ -15,16 +15,25 @@ interface FetchGroceryItemsResponseElement {
   price: number;
 }
 
-interface GroceryListItem {
+interface GroceryItem {
   id: string;
   name: string;
   price: number;
+}
+
+enum GroceryLineItemLocation {
+  List,
+  Cart,
+}
+
+interface GroceryLineItem extends GroceryItem {
```

Comment:
A `GroceryLineItem` is a `GroceryItem` with a quantity and a location (either cart or list)